torchft/process_group.py

```diff
@@ -739,7 +739,7 @@ def _worker(
             try:
                 pg = cls._create_pg(store, rank, world_size)
             except Exception as e:
-                logger.exception(f"got exception in worker: {e}")
+                print(f"got exception in worker: {e}", file=sys.stderr)
```

Comment:
any particular reason for the logger -> print change?

Reply:
We're not initializing the logger in the subprocess so switched to stderr to make sure we actually log these

Maybe I should just instantiate a second logger as well